subsys/net/lib/nrf_cloud/src/nrf_cloud_info.c

```diff
@@ -140,7 +140,16 @@ int nrf_cloud_print_details(void)
 	LOG_INF("Sec tag:   %d", nrf_cloud_sec_tag_get());
 	LOG_INF("Host name: %s", host_name);
 
-	/* Tenant will not be known unless device is connected to cloud with MQTT. */
+#endif /* CONFIG_NRF_CLOUD_VERBOSE_DETAILS */
+
+	return err;
+}
+
+int nrf_cloud_print_cloud_details(void)
```

Comment:
can this function just be called automatically by the nrf_cloud lib when the team id is obtained (if CONFIG_NRF_CLOUD_VERBOSE_DETAILS is enabled)?

Reply:
the stage can be determined by the host name, which is already printed.
i say we could just remove printing the stage since customers don't have access to anything else.

Reply:
Ha, funny you suggest just printing it in the library. I went down that road and then undid it and moved it back to where it is. This is because `nrf_cloud_print_details` is not called by the library. so printing the tenant regardless of whether the application calls the first function seemed inconsistent to me.

Suggestion?

Reply:
if the user has enabled `CONFIG_NRF_CLOUD_VERBOSE_DETAILS`, i don't think they would be upset about the team ID being automatically printed 🤷‍♂️ 

reading the kconfig description of `NRF_CLOUD_VERBOSE_DETAILS`, it sounds to me like the stuff will be printed automatically.

Reply:
What do you think about calling the `nrf_cloud_print_details` from the library as well?

Reply:
sounds like a good idea, probably should be kconfigurable.

Reply:
Done. I do need to update the changelog to reflect this still.